src/dotnet/APIView/APIView/MethodAPIV.cs

```diff
@@ -40,12 +41,15 @@ public MethodAPIV(IMethodSymbol symbol)
             {
                 this.Name = symbol.ContainingType.Name;
                 this.IsConstructor = true;
-                this.ReturnType = "";
+                this.ClassNavigationID = symbol.ContainingType.ToDisplayString();
             }
             else
             {
                 this.Name = symbol.Name;
-                this.ReturnType = symbol.ReturnType.ToString();
+                this.ReturnType = new TypeReference(symbol.ReturnType);
+                if (symbol.ReturnType.SpecialType == SpecialType.System_String)
```

Comment:
?

Reply:
These are leftover from when I was trying to implement the single token constructor - I'll toss the IsString bit when I make a push fixing attribute constructors